ci/nginx_72/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     restart: always
     image: openemr/dev-php-fpm:7.2
     environment:
-      E2E_BASE_URL: "http://nginx"
+      OPENEMR_BASE_URL: "http://nginx"
```

Comment:
I renamed this variable as I have the suspicion we will need to use it for the API tests once they are enabled.